packages/http/src/forwarder/__tests__/HttpForwarder.spec.ts

```diff
@@ -1,26 +1,25 @@
 import * as axios from 'axios';
 
-import { httpOperations, httpRequests } from '../../__tests__/fixtures';
+import { httpInputs, httpOperations, httpRequests } from '../../__tests__/fixtures';
 import { HttpForwarder } from '../HttpForwarder';
 
+jest.mock('axios', () => ({
+  default: jest.fn(async () => ({ status: 200 })),
```

Comment:
You can use `mockResolvedValue` which is a little bit cleaner

```suggestion
  default: jest.fn().mockResolvedValue({status:200})
```

Reply:
👍